ui/components/Filters.js

```diff
@@ -241,6 +281,58 @@ function MesheryFilters({ updateProgress, enqueueSnackbar, closeSnackbar, user,
     });
   }
 
+  const handlePublishModal = (ev, filter) => {
+    if (canPublishFilter) {
+      ev.stopPropagation();
+      setPublishModal({
+        open : true,
+        filter : filter
+      });
+    }
+  };
+
+  const handleUnpublishModal = (ev, filter) => {
+    if (canPublishFilter) {
+      ev.stopPropagation();
+      return async () => {
+        let response = await modalRef.current.show({
+          title : `Unpublish Catalog item?`,
+          subtitle : `Are you sure you want to unpublish ${filter?.name}?`,
+          options : ["Yes", "No"]
+        });
+        if (response === "Yes") {
+          updateProgress({ showProgress : true });
+          dataFetch(
+                `/api/filter/catalog/unpublish`,
+                { credentials : "include", method : "DELETE", body : JSON.stringify({ "id" : filter?.id }) },
+                () => {
+                  updateProgress({ showProgress : false });
+                  enqueueSnackbar("Filter unpublished", {
```

Comment:
What filter? Provide the name as confirmation to the user.

@Chadha93, remember when you had to go through the code base and make the notifications uniform by adding the name of the thing into the toaster message... ? Please uphold this convention.